README.md

```diff
@@ -444,9 +444,9 @@ https://github.com/fastfloat/fast_float/releases/download/v6.1.6/fast_float.h
 - It is part of the [brew package manager](https://formulae.brew.sh/formula/fast_float).
 - Some Linux distribution like Fedora include fast_float (e.g., as `fast_float-devel`).
 
-## RFC 7159
+## RFC 8259
```

Comment:
This remark may be obsolete and was written before we had `fast_float::chars_format::json`. I wonder whether we couldn't not replace it with something like...

```
If you need support for RFC 8259 (JSON standard), you may want to consider use the `fast_float::chars_format::json` flag
```

Or something of the sort.

Thoughts?

Reply:
Perhaps we can remove this section entirely, as `fast_float::chars_format::json` is exemplified above at line 283.

Reply:
Yes. What about that?

I would not have made this comment, but given that you are modifying this section, maybe you can just delete it.

It is fine if you don't do it, and we can use another PR to do so.

Reply:
I removed the section and updated the README.md with the new format flags as well, and some formatting changes.